contracts/BC_fusion/StakeHub.sol

```diff
@@ -430,6 +427,14 @@ contract StakeHub is System {
         emit Claimed(operatorAddress, msg.sender, bnbAmount);
     }
 
+    function sync(
```

Comment:
Please use multiple pool,  is it possible pass pool address directly, if not, I think we should maintain a map of stake pool

Reply:
fixed, modify it to `function sync(address[] calldata operatorAddresses, address account) external`